### ISSUE TITLE
Some jobs appear as waiting, but are executed and done.

### ISSUE THREAD

**Issue by ruskid** [zhuravljov/yii2-queue-monitor#3]
All jobs are executed fine and but it's just the monitor which displays some jobs as done and some as waiting.

It started after supervisor service restart. Any idea why?

**Comment by zhuravljov** [comment]
Вряд ли это связано с супервайзером. Модуль пишет статистику через обработчики событий. Это универсальный способ, не завязанный на системное окружение.

Похоже на то, что не срабатывает обработчик `JobMonitor::beforeExec()`. А вот почему именно это происходит в вашем проекте сказать сложно. Недостаточно информации.

**Comment by ruskid** [comment]
Да, скорее всего обработчик. Есть запись в *queue_push*, но нету в *queue_exec*.
Консольная статистика нормально показывает, но она привязана к редису. 

Спасибо, покопаюсь на досуге.

**Comment by zhuravljov** [comment]
Пожалуйста.
